driving_log_replayer_v2/scripts/perception_evaluator_node.py

```diff
@@ -219,23 +230,29 @@ def list_dynamic_object_from_ros_msg(
                     ),
                 ),
                 velocity=eval_conversions.velocity_from_ros_msg(
-                    perception_object.kinematics.twist_with_covariance.twist.linear,
+                    perception_object.kinematics.twist_with_covariance.twist.linear
+                    if isinstance(perception_object, DetectedObject | TrackedObject)
+                    else perception_object.kinematics.initial_twist_with_covariance.twist.linear
                 ),
-                pose_covariance=perception_object.kinematics.pose_with_covariance.covariance,
-                twist_covariance=perception_object.kinematics.twist_with_covariance.covariance,
+                pose_covariance=perception_object.kinematics.pose_with_covariance.covariance
+                if isinstance(perception_object, DetectedObject | TrackedObject)
+                else perception_object.kinematics.initial_pose_with_covariance.covariance,
+                twist_covariance=perception_object.kinematics.twist_with_covariance.covariance
+                if isinstance(perception_object, DetectedObject | TrackedObject)
+                else perception_object.kinematics.initial_twist_with_covariance.covariance,
```

Comment:
It also need to set predicted attributes for predicted object evaluation. https://github.com/tier4/autoware_perception_evaluation/blob/859b8672f63e54d83d5fc9eee66ccaa067197814/perception_eval/perception_eval/common/object.py#L112-L115

Reply:
Thank you